packages/x-codemod/src/types.ts

```diff
@@ -1,9 +1,8 @@
 import type { FileInfo, API } from 'jscodeshift';
+import { MakeOptional } from '@mui/x-internals/helpers';
 
 type MakeRequired<Type, Key extends keyof Type> = Pick<Type, Key> & Partial<Omit<Type, Key>>;
```

Comment:
This `MakeRequired` type differs from the one used in other places. 🙈

Reply:
Should we rename it `KeepRequired`? Because it does not make the key as required one, but keep them in their default definition

Reply:
Thanks for a great suggestion. 👍 
Renamed. 😉